wire-ios/Tests/TestPlans/AllTests.xctestplan

```diff
@@ -52,7 +52,6 @@
         "AVURLAsset_conversionTests\/testThatVideoIsConvertedToUploadFormat()",
         "ConversationListTopBarViewControllerSnapshotTests\/testForLongName()",
         "MessageReplyPreviewViewTests\/testThatItRendersImageMessagePreview()",
-        "ProfileViewControllerTests",
```

Comment:
Shouldn't the new tests be added to the testplan?

Reply:
I've just removed them from the skipped tests list